spec/features/punches_list_spec.rb

```diff
@@ -48,4 +51,13 @@
 
     expect(page).to have_css('tbody tr', count: 1)
   end
+
+  it "Renders attachment link" do
+    expect(page).to have_link('', href: punch_with_attachment.attachment_url)  
+  end
+
+  scenario 'follow attachment link' do
+    find("a[href='#{punch_with_attachment.attachment_url}']").click
+    expect(page).to have_content('This is a punch attachment')
```

Comment:
Couldn't find a good way to get attachment body, so i just transcribed it.
Is there a better way to do this?